### PR TITLE
[Fix][Data Quality] Change t_ds_dq_rule_input_entry field name fix PostgreSQL not support value issue

### PR DESCRIPTION
**Pull request by qingwli**
- fix #14717
- fix #14807
- fix #14653

### CODE REVIEW

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql [hunk 1]
```diff
@@ -14,3 +14,40 @@
...
+CREATE OR REPLACE FUNCTION modify_data_t_ds_dq_rule_input_entry() RETURNS void AS $$
+BEGIN
+       IF EXISTS (SELECT 1
+                  FROM information_schema.columns
+                  WHERE table_name = 't_ds_dq_rule_input_entry'
+                  AND column_name = 'value')
+      THEN
+         ALTER TABLE t_ds_dq_rule_input_entry
+                 RENAME COLUMN "value" TO "data";
+       END IF;
```
Comment (Radeity) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql]: Please align `THEN` with `IF EXISTS`

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql [hunk 2]
```diff
@@ -14,3 +14,40 @@
...
+
+-- modify_data_type_t_ds_dq_rule_input_entry
+delimiter d//
+CREATE OR REPLACE FUNCTION modify_data_type_t_ds_dq_rule_input_entry() RETURNS void AS $$
+BEGIN
+       IF EXISTS (SELECT 1
+                  FROM information_schema.columns
+                  WHERE table_name = 't_ds_dq_rule_input_entry'
+                  AND column_name = 'value_type')
+      THEN
```
Comment (Radeity) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql:55]: Ditto.

**Review by Radeity — COMMENTED** [review]
Others LGTM.

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql (hunk 2 above)
Comment (qingwli) [reply dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/postgresql/dolphinscheduler_ddl.sql:55]: Good catch and done

**Comment by qingwli** [comment]
PTAL @Radeity @zixi0825

**Review by Radeity — APPROVED** [review]

**Pull request merged**